recipes/tatc/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "tatc" %}
+{% set version = "3.1.0.beta1" %}
```

Comment:
This is not a stable release and falls under: https://github.com/conda-forge/cfep/blob/main/cfep-05.md

Reply:
Thanks for identifying this; I will work to make a full release in the coming days.

Reply:
This recipe has been updated for a stable release.